config/pipeline.yaml

```diff
@@ -101,6 +101,22 @@ storage:
 
 runtimes:
 
+  lab-kontron:
+    lab_type: legacy.lava_rest
+    url: 'https://lavalab.kontron.com/'
+    filters:
+      - passlist:
+          plan:
+            - baseline
+
+  lab-pengutronix:
+    lab_type: legacy.lava_rest
+    url: 'https://hekla.openlab.pengutronix.de/'
+    filters:
+      - passlist:
+          plan:
+            - baseline
+
```

Comment:
I realize tests for Android trees were run in those two labs only. Let's focus on getting just Android builds in the scope of this PR and move lab enablement to a separate one - it will require [very similar change](https://github.com/kernelci/kernelci-pipeline/pull/474) and some admin work (token management) on both (KCI, lab) sides.

Reply:
but to get Android builds, don't we need those labs?

Reply:
@pawiecz @a-wai please check https://github.com/kernelci/kernelci-pipeline/pull/528

Reply:
@pawiecz @a-wai  I also updated this PR to just add the kbuild jobs, please check